CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # main
 
+
+# 0.0.6
+
+* Updated the invalid class name error message
+
+# 0.0.5
```

Comment:
it looks like we already released v 0.0.5 but there's nothing in it in the changelog so I just left it blank